pkg/fsutils/fsutils_windows.go

```diff
@@ -0,0 +1,35 @@
+//go:build windows
+
+package fsutils
+
+import (
+	"os"
+	"path/filepath"
+	"syscall"
+)
+
+func evalSymlinks(path string) (string, error) {
+	resolved, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		// This is a workaround for the behavior of filepath.EvalSymlinks, which fails with
+		// syscall.ENOTDIR if the specified path contains a junction on Windows. Junctions
+		// can occur, for example, when a volume is mounted as a subdirectory inside another
+		// drive. This can usually happen when using the Dev Drives feature and replacing
+		// existing directories. See: https://github.com/golang/go/issues/40180
+		//
+		// Since syscall.ENOTDIR is only returned when calling filepath.EvalSymlinks on
+		// Windows if part of the presented path is a junction and nothing before was a
+		// symlink, we simply treat this as NOT symlink, because a symlink over the junction
+		// makes no sense at all.
+		if err == syscall.ENOTDIR {
```

Comment:
Since [`Errno`](https://pkg.go.dev/syscall#Errno), which `syscall.ENOTDIR` is, implements [`Is`](https://pkg.go.dev/syscall#Errno.Is) explicitly, should we use [the `errors.Is` comparison recommended since Go 1.13](https://go.dev/wiki/ErrorValueFAQ)?

```suggestion
		if errors.Is(err, syscall.ENOTDIR) {
```

Reply:
Yeah. I was struggling. Usually I do it always the same way. But from the perspective that this quirky workaround for something I would suspect the SDK should do better anyway, I want to be ensure to really catch the exact same error which is in this case not wrapped. But as the code might change anyway in the future, you're right. Just wrap it.